.ci/update-github-page.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+TARGET_RELEASE_NUM=$1
```

Comment:
Do validation that $1 is defined 

```
if [[ -z $1 ]]; then
  echo "release number is not set"
    exit 1
fi
```

Reply:
Added

Reply:
Put validation before this line

Reply:
I do not think it is required. It works fine

Reply:
ok, changes done